spec/order_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'order'
+
+describe Order do
+  let(:subject) { described_class.new }
```

Comment:
Just wondering what the described_class.new is referring to. It wasn't clear for me.

Reply:
described class just means the class that you are testing - in this case it is order.